### ISSUE TITLE
Application shows only blank page in release mode after upgrading cloud_firestore package from 0.13.1+1 to 0.13.2 (or higher)

### ISSUE THREAD

**Issue by takilevi** [FirebaseExtended/flutterfire#2711]
I try to use the web integration with Firestore, but when I upgrade my package to the desirable **0.13.2 from 0.13.1+1** , then the release build shows only a blank page, while debug run shows the perfect refords fetched from the database.
I've faced it developing my app, but at the end i delete everything and just "copied" the example from [cloud_firestore example section](https://pub.dev/packages/cloud_firestore#-example-tab-).

So on debug mode, and on android the list is there, loaded perfectly, on **web release build**, it shows a **blank white page**.

Some more testing examples:

- By 0.13.1+1
Android build shows the records, both web builds (debug and release) show the 'Loading...' text.
- By 0.13.2
Android still shows the records, chrome debug build shows the records, chrome release build shows blank white page.

### Flutter doctor
<details>
  <summary>flutter doctor -v</summary>

PS P:\shootingapp> flutter doctor -v
[√] Flutter (Channel beta, 1.18.0-11.1.pre, on Microsoft Windows [Version 10.0.18363.418], locale hu-HU)
    • Flutter version 1.18.0-11.1.pre at C:\flutter
    • Framework revision 2738a1148b (3 weeks ago), 2020-05-13 15:24:36 -0700
    • Engine revision ef9215ceb2
    • Dart version 2.9.0 (build 2.9.0-8.2.beta)

[√] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
    • Android SDK at C:\Users\takacsl\AppData\Local\Android\sdk
    • Platform android-28, build-tools 28.0.3
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1343-b01)
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files (x86)\Google\Chrome\Application\chrome.exe

[!] Android Studio (version 3.4)
    • Android Studio at C:\Program Files\Android\Android Studio
    X Flutter plugin not installed; this adds Flutter specific functionality.
    X Dart plugin not installed; this adds Dart specific functionality.
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1343-b01)

[√] IntelliJ IDEA Ultimate Edition (version 2019.1)
    • IntelliJ at C:\Program Files\JetBrains\IntelliJ IDEA 2019.1.3
    • Flutter plugin version 39.0.3
    • Dart plugin version 191.8593

[!] VS Code (version 1.45.1)
    • VS Code at C:\Users\takacsl\AppData\Local\Programs\Microsoft VS Code
    X Flutter extension not installed; install from
      https://marketplace.visualstudio.com/items?itemName=Dart-Code.flutter

[√] Connected device (3 available)
    • GM1913     • 99350f72   • android-arm64  • Android 10 (API 29)
    • Web Server • web-server • web-javascript • Flutter Tools
    • Chrome     • chrome     • web-javascript • Google Chrome 83.0.4103.61

! Doctor found issues in 2 categories.

</details>

### Builds
<details>
  <summary> flutter run -d chrome --release --verbose</summary>
PS P:\shootingapp> flutter run -d chrome --release --verbose
[  +10 ms] executing: [C:\flutter/] git -c log.showSignature=false log -n 1 --pretty=format:%H
[ +166 ms] Exit code 0 from: git -c log.showSignature=false log -n 1 --pretty=format:%H
[   +6 ms] 2738a1148ba6c9a6114df62358109407c3ef2553
[   +1 ms] executing: [C:\flutter/] git tag --contains HEAD
[ +335 ms] Exit code 0 from: git tag --contains HEAD
[   +2 ms] 1.18.0-11.1.pre
[  +15 ms] executing: [C:\flutter/] git rev-parse --abbrev-ref --symbolic @{u}
[ +156 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{u}
[   +2 ms] origin/beta
[   +4 ms] executing: [C:\flutter/] git ls-remote --get-url origin
[ +140 ms] Exit code 0 from: git ls-remote --get-url origin
[   +2 ms] https://github.com/flutter/flutter.git
[ +181 ms] executing: [C:\flutter/] git rev-parse --abbrev-ref HEAD
[ +145 ms] Exit code 0 from: git rev-parse --abbrev-ref HEAD
[   +2 ms] beta
[  +71 ms] Artifact Instance of 'AndroidMavenArtifacts' is not required, skipping update.
[   +2 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[   +3 ms] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[   +5 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[   +6 ms] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[   +4 ms] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[  +26 ms] executing: C:\Users\takacsl\AppData\Local\Android\sdk\platform-tools\adb.exe devices -l
[  +58 ms] List of devices attached
           99350f72               device product:OnePlus7Pro_EEA model:GM1913 device:OnePlus7Pro transport_id:3
[  +11 ms] Artifact Instance of 'AndroidMavenArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[   +6 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[   +2 ms] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[ +114 ms] Found plugin cloud_firestore at C:\flutter\.pub-cache\hosted\pub.dartlang.org\cloud_firestore-0.13.2\
[  +14 ms] Found plugin cloud_firestore_web at C:\flutter\.pub-cache\hosted\pub.dartlang.org\cloud_firestore_web-0.1.1+2\
[  +14 ms] Found plugin firebase_auth at C:\flutter\.pub-cache\hosted\pub.dartlang.org\firebase_auth-0.16.1\
[   +7 ms] Found plugin firebase_auth_web at C:\flutter\.pub-cache\hosted\pub.dartlang.org\firebase_auth_web-0.1.3+1\
[   +7 ms] Found plugin firebase_core at C:\flutter\.pub-cache\hosted\pub.dartlang.org\firebase_core-0.4.5\
[   +7 ms] Found plugin firebase_core_web at C:\flutter\.pub-cache\hosted\pub.dartlang.org\firebase_core_web-0.1.1+2\
[  +87 ms] Found plugin cloud_firestore at C:\flutter\.pub-cache\hosted\pub.dartlang.org\cloud_firestore-0.13.2\
[   +7 ms] Found plugin cloud_firestore_web at C:\flutter\.pub-cache\hosted\pub.dartlang.org\cloud_firestore_web-0.1.1+2\
[   +7 ms] Found plugin firebase_auth at C:\flutter\.pub-cache\hosted\pub.dartlang.org\firebase_auth-0.16.1\
[   +7 ms] Found plugin firebase_auth_web at C:\flutter\.pub-cache\hosted\pub.dartlang.org\firebase_auth_web-0.1.3+1\
[   +6 ms] Found plugin firebase_core at C:\flutter\.pub-cache\hosted\pub.dartlang.org\firebase_core-0.4.5\
[   +4 ms] Found plugin firebase_core_web at C:\flutter\.pub-cache\hosted\pub.dartlang.org\firebase_core_web-0.1.1+2\
[  +93 ms] Generating P:\shootingapp\android\app\src\main\java\io\flutter\plugins\GeneratedPluginRegistrant.java
[ +122 ms] Launching lib\main.dart on Chrome in release mode...
[  +74 ms] Found plugin cloud_firestore at C:\flutter\.pub-cache\hosted\pub.dartlang.org\cloud_firestore-0.13.2\
[   +4 ms] Found plugin cloud_firestore_web at C:\flutter\.pub-cache\hosted\pub.dartlang.org\cloud_firestore_web-0.1.1+2\
[   +7 ms] Found plugin firebase_auth at C:\flutter\.pub-cache\hosted\pub.dartlang.org\firebase_auth-0.16.1\
[   +6 ms] Found plugin firebase_auth_web at C:\flutter\.pub-cache\hosted\pub.dartlang.org\firebase_auth_web-0.1.3+1\
[   +2 ms] Found plugin firebase_core at C:\flutter\.pub-cache\hosted\pub.dartlang.org\firebase_core-0.4.5\
[   +5 ms] Found plugin firebase_core_web at C:\flutter\.pub-cache\hosted\pub.dartlang.org\firebase_core_web-0.1.1+2\
[  +39 ms] Found plugin cloud_firestore at C:\flutter\.pub-cache\hosted\pub.dartlang.org\cloud_firestore-0.13.2\
[   +4 ms] Found plugin cloud_firestore_web at C:\flutter\.pub-cache\hosted\pub.dartlang.org\cloud_firestore_web-0.1.1+2\
[   +7 ms] Found plugin firebase_auth at C:\flutter\.pub-cache\hosted\pub.dartlang.org\firebase_auth-0.16.1\
[   +6 ms] Found plugin firebase_auth_web at C:\flutter\.pub-cache\hosted\pub.dartlang.org\firebase_auth_web-0.1.3+1\
[   +2 ms] Found plugin firebase_core at C:\flutter\.pub-cache\hosted\pub.dartlang.org\firebase_core-0.4.5\
[   +3 ms] Found plugin firebase_core_web at C:\flutter\.pub-cache\hosted\pub.dartlang.org\firebase_core_web-0.1.1+2\
[  +32 ms] Generating P:\shootingapp\android\app\src\main\java\io\flutter\plugins\GeneratedPluginRegistrant.java
[  +15 ms] Compiling lib\main.dart for the Web...
[  +17 ms] Initializing file store
[  +19 ms] Done initializing file store
[ +101 ms] Skipping target: web_entrypoint
[ +769 ms] Skipping target: dart2js
[ +225 ms] Skipping target: web_release_bundle
[  +41 ms] Skipping target: web_service_worker
[  +17 ms] Persisting file store
[  +13 ms] Done persisting file store
[  +19 ms] Compiling lib\main.dart for the Web... (completed in 1,2s)
[ +301 ms] [CHROME]:
[  +27 ms] [CHROME]:DevTools listening on ws://127.0.0.1:55311/devtools/browser/ee04bbcf-4f68-41e3-bed5-f901da64bc1d
[ +200 ms] Warning: Flutter's support for web development is not stable yet and hasn't
[   +3 ms] been thoroughly tested in production environments.
[   +8 ms] For more information see https://flutter.dev/web
[   +1 ms]   To hot restart changes while running, press "r". To hot restart (and refresh the browser), press "R".
[   +3 ms] For a more detailed help message, press "h". To quit, press "q".
</details>

<details>
  <summary> flutter run -d chrome --debug--verbose</summary>
PS P:\shootingapp> flutter run -d chrome --debug --verbose
[  +10 ms] executing: [C:\flutter/] git -c log.showSignature=false log -n 1 --pretty=format:%H
[ +161 ms] Exit code 0 from: git -c log.showSignature=false log -n 1 --pretty=format:%H
[   +3 ms] 2738a1148ba6c9a6114df62358109407c3ef2553
[   +4 ms] executing: [C:\flutter/] git tag --contains HEAD
[ +314 ms] Exit code 0 from: git tag --contains HEAD
[   +2 ms] 1.18.0-11.1.pre
[  +14 ms] executing: [C:\flutter/] git rev-parse --abbrev-ref --symbolic @{u}
[ +137 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{u}
[   +2 ms] origin/beta
[   +4 ms] executing: [C:\flutter/] git ls-remote --get-url origin
[ +130 ms] Exit code 0 from: git ls-remote --get-url origin
[   +2 ms] https://github.com/flutter/flutter.git
[ +185 ms] executing: [C:\flutter/] git rev-parse --abbrev-ref HEAD
[ +146 ms] Exit code 0 from: git rev-parse --abbrev-ref HEAD
[   +2 ms] beta
[  +71 ms] Artifact Instance of 'AndroidMavenArtifacts' is not required, skipping update.
[   +2 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[   +3 ms] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[   +5 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[   +4 ms] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[   +2 ms] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[   +4 ms] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[  +28 ms] executing: C:\Users\takacsl\AppData\Local\Android\sdk\platform-tools\adb.exe devices -l
[  +60 ms] List of devices attached
           99350f72               device product:OnePlus7Pro_EEA model:GM1913 device:OnePlus7Pro transport_id:3
[  +11 ms] Artifact Instance of 'AndroidMavenArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[   +5 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[   +4 ms] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[   +3 ms] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[   +3 ms] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[ +113 ms] Found plugin cloud_firestore at C:\flutter\.pub-cache\hosted\pub.dartlang.org\cloud_firestore-0.13.2\
[  +12 ms] Found plugin cloud_firestore_web at C:\flutter\.pub-cache\hosted\pub.dartlang.org\cloud_firestore_web-0.1.1+2\
[  +14 ms] Found plugin firebase_auth at C:\flutter\.pub-cache\hosted\pub.dartlang.org\firebase_auth-0.16.1\
[  +10 ms] Found plugin firebase_auth_web at C:\flutter\.pub-cache\hosted\pub.dartlang.org\firebase_auth_web-0.1.3+1\
[   +4 ms] Found plugin firebase_core at C:\flutter\.pub-cache\hosted\pub.dartlang.org\firebase_core-0.4.5\
[   +6 ms] Found plugin firebase_core_web at C:\flutter\.pub-cache\hosted\pub.dartlang.org\firebase_core_web-0.1.1+2\
[  +79 ms] Found plugin cloud_firestore at C:\flutter\.pub-cache\hosted\pub.dartlang.org\cloud_firestore-0.13.2\
[   +7 ms] Found plugin cloud_firestore_web at C:\flutter\.pub-cache\hosted\pub.dartlang.org\cloud_firestore_web-0.1.1+2\
[   +7 ms] Found plugin firebase_auth at C:\flutter\.pub-cache\hosted\pub.dartlang.org\firebase_auth-0.16.1\
[   +5 ms] Found plugin firebase_auth_web at C:\flutter\.pub-cache\hosted\pub.dartlang.org\firebase_auth_web-0.1.3+1\
[   +7 ms] Found plugin firebase_core at C:\flutter\.pub-cache\hosted\pub.dartlang.org\firebase_core-0.4.5\
[   +4 ms] Found plugin firebase_core_web at C:\flutter\.pub-cache\hosted\pub.dartlang.org\firebase_core_web-0.1.1+2\
[  +82 ms] Generating P:\shootingapp\android\app\src\main\java\io\flutter\plugins\GeneratedPluginRegistrant.java
[ +126 ms] Launching lib\main.dart on Chrome in debug mode...
[  +83 ms] Updating assets
[ +128 ms] Syncing files to device Chrome...
[  +14 ms] Found plugin cloud_firestore at C:\flutter\.pub-cache\hosted\pub.dartlang.org\cloud_firestore-0.13.2\
[   +3 ms] Found plugin cloud_firestore_web at C:\flutter\.pub-cache\hosted\pub.dartlang.org\cloud_firestore_web-0.1.1+2\
[   +5 ms] Found plugin firebase_auth at C:\flutter\.pub-cache\hosted\pub.dartlang.org\firebase_auth-0.16.1\
[   +4 ms] Found plugin firebase_auth_web at C:\flutter\.pub-cache\hosted\pub.dartlang.org\firebase_auth_web-0.1.3+1\
[   +4 ms] Found plugin firebase_core at C:\flutter\.pub-cache\hosted\pub.dartlang.org\firebase_core-0.4.5\
[   +4 ms] Found plugin firebase_core_web at C:\flutter\.pub-cache\hosted\pub.dartlang.org\firebase_core_web-0.1.1+2\
[  +32 ms] Found plugin cloud_firestore at C:\flutter\.pub-cache\hosted\pub.dartlang.org\cloud_firestore-0.13.2\
[   +5 ms] Found plugin cloud_firestore_web at C:\flutter\.pub-cache\hosted\pub.dartlang.org\cloud_firestore_web-0.1.1+2\
[   +6 ms] Found plugin firebase_auth at C:\flutter\.pub-cache\hosted\pub.dartlang.org\firebase_auth-0.16.1\
[   +3 ms] Found plugin firebase_auth_web at C:\flutter\.pub-cache\hosted\pub.dartlang.org\firebase_auth_web-0.1.3+1\
[   +3 ms] Found plugin firebase_core at C:\flutter\.pub-cache\hosted\pub.dartlang.org\firebase_core-0.4.5\
[   +4 ms] Found plugin firebase_core_web at C:\flutter\.pub-cache\hosted\pub.dartlang.org\firebase_core_web-0.1.1+2\
[  +28 ms] Generating P:\shootingapp\android\app\src\main\java\io\flutter\plugins\GeneratedPluginRegistrant.java
[  +56 ms] <- reset
[  +12 ms] C:\flutter\bin\cache\dart-sdk\bin\dart.exe C:\flutter\bin\cache\artifacts\engine\windows-x64\frontend_server.dart.snapshot --sdk-root C:\flutter\bin\cache\flutter_web_sdk/
--incremental --target=dartdevc --debugger-module-names -Ddart.developer.causal_async_stacks=true --output-dill
C:\Users\takacsl\AppData\Local\Temp\flutter_tool.87076396-a81b-11ea-ad0e-34f39a073ad1\app.dill --libraries-spec file:///C:/flutter/bin/cache/flutter_web_sdk/libraries.json --packages
.packages -Ddart.vm.profile=false -Ddart.vm.product=false
--bytecode-options=source-positions,local-var-info,debugger-stops,instance-field-initializers,keep-unreachable-code,avoid-closure-call-instructions --enable-asserts --track-widget-creation
--filesystem-root C:\Users\takacsl\AppData\Local\Temp\flutter_tools.87076397-a81b-11ea-ad0e-34f39a073ad1 --filesystem-scheme org-dartlang-app --initialize-from-dill build\cache.dill
--platform file:///C:/flutter/bin/cache/flutter_web_sdk/kernel/flutter_ddc_sdk.dill
[ +133 ms] <- compile org-dartlang-app:/web_entrypoint.dart
[+18730 ms] Syncing files to device Chrome... (completed in 19 037ms, longer than expected)
[   +3 ms] Synced 23.5MB.
[   +2 ms] <- accept
[ +267 ms] [CHROME]:
[  +23 ms] [CHROME]:DevTools listening on ws://127.0.0.1:55663/devtools/browser/921da48f-a708-4c57-a4f8-083409eecd2b
[+5067 ms] Debug service listening on ws://127.0.0.1:55694/-ZfUfZCJcCE=

[ +658 ms] Debug service listening on ws://127.0.0.1:55694/-ZfUfZCJcCE=
[   +9 ms] Warning: Flutter's support for web development is not stable yet and hasn't
[   +1 ms] been thoroughly tested in production environments.
[   +2 ms] For more information see https://flutter.dev/web
[   +2 ms]   To hot restart changes while running, press "r". To hot restart (and refresh the browser), press "R".
[   +3 ms] For a more detailed help message, press "h". To quit, press "q".
</details>

### Config
<details>
  <summary>pubspec.yaml</summary>
environment:
  sdk: ">=2.7.0 <3.0.0"

dependencies:
  flutter:
    sdk: flutter
  provider: ^4.1.2
  firebase_core: ^0.4.4
  firebase_auth: ^0.16.1
  cloud_firestore: 0.13.2
  cupertino_icons: ^0.1.3

dev_dependencies:
  flutter_test:
    sdk: flutter
</details>

**Comment by TahaTesser** [comment]
Hi @takilevi 
Tried the official [example](https://pub.dev/packages/cloud_firestore#-example-tab-) with latest `cloud_firestore: ^0.13.6`,no issues in release mode 

<img width="488" alt="Screenshot 2020-06-08 at 7 58 17 PM" src="https://user-images.githubusercontent.com/48603081/84042377-9c583f00-a9c2-11ea-8740-2323e0390634.png">

<details> 
<summary>flutter doctor -v</summary> 

``` 
[✓] Flutter (Channel dev, 1.19.0-4.0.pre, on Mac OS X 10.15.5 19F101, locale en-GB)
    • Flutter version 1.19.0-4.0.pre at /Users/tahatesser/Code/flutter_dev
    • Framework revision 2f7a59a8da (3 days ago), 2020-06-05 03:44:02 -0700
    • Engine revision d17c84e7af
    • Dart version 2.9.0 (build 2.9.0-13.0.dev 02915ec5ce)

 
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.3)
    • Android SDK at /Users/tahatesser/Code/sdk
    • Platform android-29, build-tools 29.0.3
    • ANDROID_HOME = /Users/tahatesser/Code/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.5)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.5, Build version 11E608c
    • CocoaPods version 1.9.3

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 4.0)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 46.0.2
    • Dart plugin version 193.7361
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)

[✓] VS Code (version 1.45.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.11.0

[✓] Connected device (4 available)
    • SM M305F   • 32003c30dc19668f • android-arm64  • Android 10 (API 29)
    • macOS      • macOS            • darwin-x64     • Mac OS X 10.15.5 19F101
    • Web Server • web-server       • web-javascript • Flutter Tools
    • Chrome     • chrome           • web-javascript • Google Chrome 83.0.4103.97

• No issues found!
``` 

</details>

**Comment by takilevi** [comment]
Yea, I did not create a new project or something, just wired out all my other files, and implement the example code in it. This is why i opened this ticket that "whether there is some hidden dependency, or compilation error that the framework may hide or swallow, which causes this symptom?"

I'm looking for some advice which leads me to not recreate my project from scratch. :(

**Comment by TahaTesser** [comment]
@takilevi 
Since you mentioned you''re facing this issue using official example and i cannot reproduce such issue

Can you please upgrade package to latest version and try again?

**Comment by takilevi** [comment]
A applied you suggestion to upgrade to `^0.13.6`, it did not help.
But I create a branch about the example what I've talked.

Here it is: https://github.com/takilevi/ShootingApp/tree/reproduction-example-%232711

`main.dart` contains only the main() function with the navigation.
`home_page.dart` contains the listview, filled from my database using users collection.

**Comment by Salakar** [comment]
Hey, as part of our on-going work for #2582, this has been resolved in our Firebase Firestore rework (#2913) - which has now been merged into master. We'll look at publishing some prereleases in the next few days. Thank you
